### PR TITLE
Added luacheck

### PR DESCRIPTION
**Pull request by Seadragon91**
This adds luacheck.
Will then look to fix a few of the listed problems, e.g. white spaces and accessing undefined variable.

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -0,0 +1,15 @@
...
+addons:
+  apt:
+    packages:
+    - lua5.1
+    - luarocks
+
+install:
+  - luarocks --local install luafilesystem
+  - luarocks --local install luacheck
+  - wget -O $TRAVIS_BUILD_DIR/.luacheckrc --no-check-certificate https://builds.cuberite.org/job/Cuberite%20Windows%20x64%20Master/lastSuccessfulBuild/artifact/Server/.luacheckrc
```
Comment (NiLSPACE) [inline .travis.yml]: I don't have any experience with this, but you're trying to use a file called `luacheckrc` while the file you've added is called `luacheckrc_plugin`. Is this correct?

Comment (Seadragon91) [inline .travis.yml]: LuaCheck will run the file `.luacheckrc` and this file contains code that will load the file `.luacheckrc_plugin` if existent. Tables in both files will be merged and and options added or overwritten.

Ref.: [main_APIDump.lua](https://github.com/cuberite/cuberite/blob/7922e6addb06de89cc73c64d30321aa6710e30ce/Server/Plugins/APIDump/main_APIDump.lua#L1705)
Build run: https://travis-ci.org/Seadragon91/WorldEdit/builds/240742891

**Comment by madmaxoft** [comment]
Before merging, you should set up the CI and have it report success :)

**Comment by Seadragon91** [comment]
Enabled CI.

**Comment by madmaxoft** [comment]
I'd remove them.

**Comment by Seadragon91** [comment]
Okay.

**Pull request merged**